src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/filters/suppressionsinglefilter/Example1.java

```diff
@@ -0,0 +1,35 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module
+      name="com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocStyleCheck"/>
```

Comment:
We need have no fully qualified names here, please recheck other PR for enabling examples for filters, there is map in code that can keep mapping of short name to full qualified